build/configs/rtl8721csm/Make.defs

```diff
@@ -61,9 +61,9 @@ LDSCRIPT = rlx8721d_img2_is.ld
 endif
 
 ifeq ($(CONFIG_LIBCXX_HAVE_LIBSUPCXX),y)
-LIBSUPXX = ${shell $(CC) $(CXXFLAGS) --print-file-name=libsupc++.a}
+LIBSUPXX = ${shell $(CC) $(CXXFLAGS) --print-file-name=libsupc++_nano.a}
 EXTRA_LIBPATHS += -L "${shell dirname "$(LIBSUPXX)"}"
-EXTRA_LIBS += -lsupc++
+EXTRA_LIBS += -lsupc++_nano
```

Comment:
other configs were using `-lsupc++`,
Could you please add commit comment why changed this?

Reply:
lsupc++_nano doesnot execute the exception libraries and for loadable we donot need exeption ig